Models/ResourceModelMap.cpp

```diff
@@ -129,8 +129,9 @@ void ResourceModelMap::ResourceRemoved(TypeCase type, const QString& name,
     }
   }
 
-  // Remove an references to this resource
-  emit ResourceRenamed(ResTypeAsString(type), name, "");
+  // Remove references to this resource
+  _resources[type].remove(name);
+  emit DataChanged();
```

Comment:
Okay. I understand what this is for, now; the original line was a hack. The main purpose of the ResourceRenamed call, from that comment, seems to have been to update code and resource references to the resource. So if you delete `sprite0` but `object0` had it set as its own sprite, the reference is updated to be the empty string. I guess we can assume that `DataChanged` will accomplish that, but be prepared to spot this regression.